Tests/VisualRecognitionV3Tests/VisualRecognitionTestUtilities.swift

```diff
@@ -37,7 +37,7 @@ let signURL = "https://raw.githubusercontent.com/watson-developer-cloud/java-sdk
 
 func loadResource(name: String, ext: String) -> Data {
     #if os(Linux)
-    let url = URL(fileURLWithPath: "Tests/VisualRecognitionV3Tests/Resources/" + name + "." + ext)
+    let url = URL(fileURLWithPath: "Tests/VisualRecognitionV4Tests/Resources/" + name + "." + ext)
```

Comment:
Is this accurate? looks like its in v3 directory

Reply:
Whoops - I was juggling a couple branches and stashed this fix... It's updated now and this file should no longer be in the PR.